models/mart/average_revenue.sql

```diff
@@ -0,0 +1,44 @@
+with customer_count as 
+	(select 
+		date_trunc('month', "date")::date as mrr_month, 
+		count(distinct customer_id) as customers,

```

Comment:
Maybe we can use a more meaningful name?

```suggestion
		count(distinct customer_id) as customers_count,

```